includes/data-stores/class-wc-customer-data-store.php

```diff
@@ -374,14 +374,14 @@ public function get_total_spent( &$customer ) {
 	 * @return array
 	 */
 	public function search_customers( $term, $limit = '' ) {
-		$query = new WP_User_Query( array(
+		$query = new WP_User_Query( apply_filters( 'woocommerce_customer_search_customers', array(
 			'search'         => '*' . esc_attr( $term ) . '*',
 			'search_columns' => array( 'user_login', 'user_url', 'user_email', 'user_nicename', 'display_name' ),
 			'fields'         => 'ID',
 			'number'         => $limit,
-		) );
+		), $term, $limit, 'main_query' );
```

Comment:
```Parse error: syntax error, unexpected ';' in /home/travis/build/woocommerce/woocommerce/includes/data-stores/class-wc-customer-data-store.php on line 382```

You have 3 opening parentheses but only 2 closing parentheses. The other query has the same issue.